site/bot/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+from bot.api import views
+
+urlpatterns = [
```

Comment:
All these URLs should only be accessible by managers, not every user, right? Please add these checks to the views.